packages/combobox/test/lion-combobox.test.js

```diff
@@ -543,6 +543,128 @@ describe('lion-combobox', () => {
       expect(options[3]).lightDom.to.equal(`Victoria Plum`);
     });
 
+    describe('Active index behavior', () => {
+      it('sets the active index to the closest match on open by default', async () => {
+        const el = /** @type {LionCombobox} */ (await fixture(html`
+          <lion-combobox name="foo">
+            <lion-option .choiceValue="${'Artichoke'}">Artichoke</lion-option>
+            <lion-option .choiceValue="${'Chard'}">Chard</lion-option>
+            <lion-option .choiceValue="${'Chicory'}">Chicory</lion-option>
+            <lion-option .choiceValue="${'Victoria Plum'}">Victoria Plum</lion-option>
+          </lion-combobox>
+        `));
+        mimicUserTyping(/** @type {LionCombobox} */ (el), 'cha');
+        await el.updateComplete;
+        expect(el.activeIndex).to.equal(1);
+      });
+
+      it('changes whether active index is set to the closest match automatically depending on autocomplete', async () => {
+        /**
+         * Automatic selection (setting activeIndex to closest matching option) in lion is set for inline & both autocomplete,
+         * because it is unavoidable there
+         * For list & none autocomplete, it is turned off and manual selection is required.
+         * TODO: Make this configurable for list & none autocomplete?
+         */
+        const el = /** @type {LionCombobox} */ (await fixture(html`
+          <lion-combobox name="foo" autocomplete="none">
+            <lion-option .choiceValue="${'Artichoke'}">Artichoke</lion-option>
+            <lion-option .choiceValue="${'Chard'}">Chard</lion-option>
+            <lion-option .choiceValue="${'Chicory'}">Chicory</lion-option>
+            <lion-option .choiceValue="${'Victoria Plum'}">Victoria Plum</lion-option>
+          </lion-combobox>
+        `));
+        // https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.1pattern/grid-combo.html
```

Comment:
We only implement the listbox right now. Maybe in the future we can support a grid as popup, but it would need to be a different component (based on a 'GridMixin').

I do see the behavior here indeed : https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.1pattern/listbox-combo.html
In example 2, the matching option gets aria-selected="true", but aria-activedescendant is still empty. A bit weird imho, since a new `ArrowDown` key activates the sibling after selected, indicating the prev activedescendant was equal to aria-selected=true.
Also, the puts a class 'focused' on the element. Since active === focused, I'm not sure if th behavior from w3c example is exactly as intended...